XenAdmin/Wizards/ExportWizard/ExportOptionsPage.cs

```diff
@@ -114,21 +114,21 @@ public ExportOptionsPage()
 		/// <summary>
 		/// Gets the page's title (headline)
 		/// </summary>
-		public override string PageTitle { get { return Messages.EXPORT_OPTIONS_PAGE_TITLE; } }
+		public override string PageTitle => Messages.EXPORT_OPTIONS_PAGE_TITLE;
 
 		/// <summary>
 		/// Gets the page's label in the (left hand side) wizard progress panel
 		/// </summary>
-		public override string Text { get { return Messages.EXPORT_OPTIONS_PAGE_TEXT; } }
+		public override string Text => Messages.EXPORT_OPTIONS_PAGE_TEXT;
 
 		/// <summary>
 		/// Gets the value by which the help files section for this page is identified
 		/// </summary>
-		public override string HelpID { get { return "ExportOptions"; } }
+		public override string HelpID => "ExportOptions";
 
         protected override void PageLoadedCore(PageLoadedDirection direction)
 		{
-			SetButtonNextEnabled(true);
+			SetButtonNextEnabled(!IsDirty || m_isEncryptionOk && m_isSignatureOk);

```

Comment:
I'm a little confused by this I would have expected:
`SetButtonNextEnabled(IsDirty && m_isEncryptionOk && m_isSignatureOk);`

Reply:
The very first time the user lands on the page they have not changed anything yet, hence IsDirty is false, however, it is valid to proceed without manifest or signature. Only if they have touched something need we check the other values.